src/grpc_requests/__init__.py

```diff
@@ -1,4 +1 @@
-from .aio import AsyncClient, ReflectionAsyncClient, StubAsyncClient, get_by_endpoint as async_get_by_endpoint
-from .client import Client, ReflectionClient, StubClient, get_by_endpoint
-
```

Comment:
If this is cleared, won't it break backwards compatibility when importing?
If it was deleted because it was determined to be unused code during the formatting check, I think you should set this file to ignore.

Reply:
Fair catch - I'll get that change pushed up ASAP